typescript/agentkit/src/action-providers/messari/README.md

```diff
@@ -0,0 +1,101 @@
+# Messari Action Provider
+
+The Messari Action Provider enables AI agents to query the [Messari AI toolkit](https://messari.io/) for crypto market research data. This provider allows agents to ask research questions about market data, statistics, rankings, historical trends, and information about specific protocols, tokens, or platforms.
+
+## Getting an API Key
+
+To use the Messari Action Provider, you need to obtain a Messari API key by following these steps:
```

Comment:
Can we add a sentence to reference the API docs about Authentication for more in-depth information?

https://docs.messari.io/reference/authentication